common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+      "error",
+      {
+        "cases": {
+          "camelCase": true,
+          "pascalCase": true
+        }
+      }
+    ],
+    "unicorn/import-index": [
+      "off"
+    ],
+    "unicorn/import-style": [
+      "error"
+    ],
+    "unicorn/new-for-builtins": [
+      "error"
+    ],
+    "unicorn/no-abusive-eslint-disable": [
+      "error"
+    ],
+    "unicorn/no-array-callback-reference": [
+      "error"
+    ],
+    "unicorn/no-array-for-each": [
+      "error"
+    ],
+    "unicorn/no-array-method-this-argument": [
+      "error"
+    ],
+    "unicorn/no-array-push-push": [
+      "error"
+    ],
+    "unicorn/no-array-reduce": [
+      "error"
+    ],
+    "unicorn/no-await-expression-member": [
+      "error"
+    ],
+    "unicorn/no-console-spaces": [
+      "error"
+    ],
+    "unicorn/no-document-cookie": [
+      "error"
+    ],
+    "unicorn/no-empty-file": [
+      "error"
+    ],
+    "unicorn/no-for-loop": [
+      "error"
+    ],
+    "unicorn/no-hex-escape": [
+      "error"
+    ],
+    "unicorn/no-instanceof-array": [
+      "error"
+    ],
+    "unicorn/no-invalid-remove-event-listener": [
+      "error"
+    ],
+    "unicorn/no-keyword-prefix": [
+      "off"
+    ],
+    "unicorn/no-lonely-if": [
+      "error"
+    ],
+    "no-nested-ternary": [
+      "off"
+    ],
+    "unicorn/no-nested-ternary": [
+      "error"
+    ],
+    "unicorn/no-new-array": [
+      "error"
+    ],
+    "unicorn/no-new-buffer": [
+      "error"
+    ],
+    "unicorn/no-null": [
+      "error"
+    ],
+    "unicorn/no-object-as-default-parameter": [
+      "error"
+    ],
+    "unicorn/no-process-exit": [
+      "error"
+    ],
+    "unicorn/no-static-only-class": [
+      "error"
+    ],
+    "unicorn/no-thenable": [
+      "error"
+    ],
+    "unicorn/no-this-assignment": [
+      "error"
+    ],
+    "unicorn/no-unreadable-array-destructuring": [
+      "error"
+    ],
+    "unicorn/no-unsafe-regex": [
+      "off"
+    ],
+    "unicorn/no-unused-properties": [
+      "off"
+    ],
+    "unicorn/no-useless-fallback-in-spread": [
+      "error"
+    ],
+    "unicorn/no-useless-length-check": [
+      "error"
+    ],
+    "unicorn/no-useless-promise-resolve-reject": [
+      "error"
+    ],
+    "unicorn/no-useless-spread": [
+      "error"
+    ],
+    "unicorn/no-useless-undefined": [
+      "error"
+    ],
+    "unicorn/no-zero-fractions": [
+      "error"
+    ],
+    "unicorn/number-literal-case": [
+      "error"
+    ],
+    "unicorn/numeric-separators-style": [
+      "error"
+    ],
+    "unicorn/prefer-add-event-listener": [
+      "error"
+    ],
+    "unicorn/prefer-array-find": [
+      "error"
+    ],
+    "unicorn/prefer-array-flat": [
+      "error"
+    ],
+    "unicorn/prefer-array-flat-map": [
+      "error"
+    ],
+    "unicorn/prefer-array-index-of": [
+      "error"
+    ],
+    "unicorn/prefer-array-some": [
+      "error"
+    ],
+    "unicorn/prefer-at": [
+      "off"
+    ],
```

Comment:
Consider enabling this: <https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-at.md>